pkg/workload/workload.go

```diff
@@ -435,6 +436,8 @@ func Split(ctx context.Context, db *gosql.DB, table Table, concurrency int) erro
 
 	log.Infof(ctx, `starting %d splits`, len(splitPoints))
 	g := ctxgroup.WithContext(ctx)
+	// Rate limit splitting to prevent replica imbalance.
+	r := rate.NewLimiter(128, 64)
```

Comment:
What's the point of specifying a burst size?

Reply:
My original idea was to not slow down small numbers of splits. I had initially also picked a much lower rate. Mostly just picked a number without much thinking and didn't revisit it. It's now 0.

Reply:
Burst needs to be at least 1.